src/background/browserAction.ts

```diff
@@ -72,6 +74,18 @@ async function _toggleSidebar(tabId: number, tabUrl: string): Promise<void> {
 }
 
 async function handleBrowserAction(tab: Tabs.Tab): Promise<void> {
+  const url = String(tab.url);
+
+  const optionsPage = browser.runtime.getURL("options.html");
+
+  if (url.startsWith(optionsPage)) {
+    const keyboardShortcut = isMac() ? "Cmd+Opt+C" : "Ctrl+Shift+C";
+    notify.info(
+      { tabId: tab.id, page: "any" },
```

Comment:
@fregante Is "any" ok here? Ran into an issue where the message would get "forwarded" if page was not specified.

Reply:
Try `page: "/options.html"` but it _should_ work even without it 🤔 

"Any" broadcasts the message to all chrome-extension:// pages and the tabId is currently ignored. This means even the background page will get the message and throw an error.

Reply:
Indeed we never received `tabId`-only messages outside the content script so it's not supported:

- https://github.com/pixiebrix/webext-messenger/issues/86

We still need to change this or else `tabId` is ignored. I tried this and it works:

```suggestion
      { tabId: tab.id, page: "/options.html" },
```